package.json

```diff
@@ -26,7 +26,8 @@
     "test:coverage": "jest --collectCoverageFrom '[\"src/components/**/*.js\", \"src/hooks/**/*.js\"]' --coverage && open ./coverage/lcov-report/index.html",
     "test:debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand",
     "test:update": "jest --updateSnapshot",
-    "test:scss": "./scss-test.sh"
+    "test:scss": "./scss-test.sh",
+    "test:types": "tsc --jsx react --noEmit -p ./src/components"
```

Comment:
This allows us to do `yarn test:types` and if no typescript errors, we've verified our typescript defs are working properly."